clover.rb

```diff
@@ -142,17 +142,13 @@ def self.freeze
     secret: Config.clover_session_secret
 
   autoload_normal("serializers/web", include_first: true)
-
-  # YYY: It'd be nice to use autoload, but it can't work while
-  # constants used across files are defined inside routes files and
-  # the autoload dependency cannot be tracked cheaply.
-  #
-  # if Unreloader.autoload?
-  #   plugin :autoload_hash_branches
-  #   autoload_hash_branch_dir("./routes")
-  # end
-
-  Unreloader.require("routes", delete_hook: proc { |f| hash_branch(File.basename(f).delete_suffix(".rb")) }) {}
+  if Unreloader.autoload?
```

Comment:
This is always true now, even in production, though the autoloads are forced to de-lazify in loader.rb. The reason for that is if autoload is converted to "require" (as Unreloader does when autoloading is disabled), load order and over-broad loading issues come back to bite, but only when the production flag is set to true.